src/main/java/frc/robot/robot_manager/RobotManager.java

```diff
@@ -261,6 +275,13 @@ public void stopIntakingRequest() {
     }
   }
 
+  public void stopIntakeAssistRequest() {
```

Comment:
this function and the `stopIntakingRequest()` should be removed, we can just use `stowRequest()`